src/v/cluster/partition_manager.cc

```diff
@@ -82,6 +83,15 @@ ss::future<> partition_manager::remove(const model::ntp& ntp) {
       .finally([partition] {}); // in the end remove partition
 }
 
+ss::future<> partition_manager::shutdown_all() {
+    std::vector<model::ntp> ntps;
+    ntps.reserve(_ntp_table.size());
+
+    for (const auto& ntp : ntps) {
+        co_await shutdown(ntp);
+    }
```

Comment:
what am I missing here? this looks like a no-op

Reply:
you are right, I was quickly refactoring this code and missed `std::transform`